karma.conf.js

```diff
@@ -85,6 +89,7 @@ module.exports = function(config) {
       "/tests/": "/base/tests/",
       "/spec/": "/base/tests/spec/",
       "/node_modules/": "/base/node_modules/",
+      "/builds/": "/base/builds/",
```

Comment:
> Hmm - maybe I don't understand the karma testing strategy. It builds the built version as a first step. 

Sorry, let me clarify. That's not Karma, that's `npm test`. "npm test" is our battery of tests, one of which is running Karma. 

> Why does it do that if it is not going to test it.

It does test the build, by running it through PhantomJS. 

>  Also, while I agree that testing a static version is different than testing the dynamic version, we don't /ship/ the dynamic version.

Sure, but trying to test the generated version makes it hard for us to debug... however, as we now have source maps (only recently added), we **could** switch to just testing the built version (*terms and conditions apply).  

>  If they test differently it is definitely a problem, but it is also a problem if we do all our testing against something we don't ship. What am I missing?

Not much. We can try switching to the built one, but we need to make sure the source maps actually work. It doesn't help us debug things if we see errors like "ReSpec threw an error on line 1 character 11232" or an uglified code-base.